sources/install/package_ad.sh

```diff
@@ -260,7 +260,7 @@ function install_impacket() {
     if [[ "$(date +%Y%m%d)" -gt "$(date -d $temp_fix_limit +%Y%m%d)" ]]; then
       criticalecho "Temp fix expired. Exiting."
     else
-      pipx inject impacket pycryptodome
+      pipx inject impacket
```

Comment:
the whole temp fix structure can be removed. `pipx inject impacket` without additional positional arguments will do nothing